lib/coarse_op.cuh

```diff
@@ -939,6 +745,8 @@ namespace quda {
       switch (type) {
       case COMPUTE_VUV:
 	Y_atomic.restore();
+	X_atomic.restore();
```

Comment:
This wasn't a bug in develop / 1.1.  It was an addition here due to how the `switch / case` statements are now written to avoid compiler warnings from `-Wextra`.